src/DAMA/DoctrineTestBundle/Doctrine/DBAL/AbstractStaticDriverV2.php

```diff
@@ -19,7 +19,7 @@ public function connect(array $params, $username = null, $password = null, array
             return $this->underlyingDriver->connect($params, $username, $password, $driverOptions);
         }
 
-        $key = sha1(serialize($params).$username.$password);
+        $key = sha1(json_encode($params).$username.$password);
```

Comment:
I think then we should add `ext-json` into the composer requirements?

Reply:
At least until we drop support for PHP < 8

Reply:
yes would be nice to somehow come up with a testcase inside `StaticDriverTest`. There you could probably add some platform instance/mock into the parameters